pallets/parachain-staking/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "parachain-staking"
-version = "1.0.3"
+version = "1.0.4"
```

```suggestion
version = "2.0.0"
```

This removes a public interface so this has to be a major bump according to semver.